src/api/aptos.ts

```diff
@@ -46,6 +50,14 @@ export class Aptos {
 
   readonly transactionSubmission: TransactionSubmission;
 
+  readonly generate: Generate;
+
+  readonly sign: Sign;
+
+  readonly simulate: Simulate;
+
+  readonly submit: Submit;
+
```

Comment:
The best part about the namespacing here is that it intuitively encourages developers to understand the different parts of building a signed transaction without being overwhelming or convoluted.

Before we half tried to do it for them, half expected them to understand- so it was confusing/unintuitive. 

I think the main thing left is to see what the multi-agent flow is like and see if other people appreciate this flow better.

Reply:
multi-agent flow

https://github.com/aptos-labs/aptos-ts-sdk/assets/29798064/bfa294d2-c8f2-4e5c-8a7d-32850f44997e